woocommerce/woocommerce
# Update Suggested Privacy Policy text to use the non-deprecated class.

## All Submissions:






### Changes proposed in this Pull Request:

Update Suggested Privacy Policy text to utilize `privacy-policy-tutorial` css class instead of `wp-policy-help` as it was deprecated. Also wrap the contents in the `wp-suggested-text` div to style the section to match WordPress. This follows from [WPCoreTrac#49282](https://core.trac.wordpress.org/ticket/49282) and although back-compat is being introduced in WP5.4 as of [WPChangeset#47112](https://core.trac.wordpress.org/changeset/47112) this change will better support users of WP5.1-5.4

### How to test the changes in this Pull Request:

1. Install updated WooCommerce on WP between 5.1 and 5.3
2. Navigate to the Privacy Policy Guide in WordPress (`/wp-admin/privacy-policy-guide.php`)
3. Confirm the styling of the guide and the copy action only grabs the appropriate contents.

### Other information:




### Changelog entry

Updated Privacy Policy Guide Suggested text to support the `privacy-policy-tutorial` and `wp-suggested-text` classes added in WP5.1 and deprecates use of `wp-policy-help` class.
